mysite/blog/tests/test_models.py

```diff
@@ -31,3 +31,7 @@ def test_author(self):
         post = Post.objects.get(id=1)
         author = post.author
         self.assertEqual(author.username, "karthikeyan")
+
+    def test_published(self):

```

Comment:
```suggestion
    def test_published_should_return_only_published_posts(self):

```

- Create a draft post and a published post
- assert Post.published.all() should have only the published post and not the draft post